child/child.go

```diff
@@ -157,6 +171,29 @@ func (c *Child) Pid() int {
 	return c.pid()
 }
 
+// Pgid returns the process group ID of the child process
+func (c *Child) Pgid() int {
+	c.RLock()
+	defer c.RUnlock()
+
+	pgid, err := syscall.Getpgid(c.cmd.Process.Pid)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	return pgid
+}
+
+// Sid returns the session ID of the child process
+func (c *Child) Sid() int {
+	c.RLock()
+	defer c.RUnlock()
+	sid, err := unix.Getsid(c.cmd.Process.Pid)
```

Comment:
Use of x/sys/unix isn't cross platform and while we don't officially support windows at this point, I don't like to purposely break it. Would you please consider refactoring this into an OS specific file with a stub/dummy version for windows.

Sorry I missed this earlier when I was checking the SysProcAttr use... not sure how I missed it.

Thanks.